libs/stokesian_dynamics/lubrication_data.inl

```diff
@@ -0,0 +1,285 @@
+static constexpr double rsabc[] = {
```

Comment:
@jngrad Do we have a policy on c-style arrays?

Reply:
We used to have a Clang-Tidy 6 check for C-style arrays, but it was not detecting all of them. When we moved to Clang-Tidy 9 a few days ago, all of them were found. I replaced some by `Utils::Vector` (b30f49f3, b848ef9f6, ce0b0f137), but there were too many of them, especially in CUDA code which hadn't been checked by Clang-Tidy 6 for a while. I eventually removed the check (994e993eb). We also can't always use `std::array` because CUDA code requires the `__device__` keyword.

Reply:
No but we can use ```Utils::array``` for cuda code, right?

Reply:
Indeed, `Utils::array` would be appropriate here.

Reply:
`Utils::array` is not appropriate as libs/stokesian_dynamics is supposed to remain an independent library. As such, it cannot include any Espresso headers. Some variant of `thrust_wrapper::device_array` might work.

Reply:
Those numbers must be available on a Thrust Device. I'm not sure if there is such a thing as a `constexpr device_vector`, haven't found anything promising on the web yet.